client-gateway/bot/passenger.py

```
@@ -1,9 +1,9 @@
-import logging
 import re
+import logging
```

Flake8 (7.3.0)

[error] 'logging' imported but unused

(F401)